standardtracer/span.go

```diff
@@ -11,22 +11,33 @@ import (
 // Implements the `Span` interface. Created via tracerImpl (see
 // `standardtracer.New()`).
 type spanImpl struct {
-	lock     sync.Mutex
-	tracer   *tracerImpl
-	recorder SpanRecorder
-	raw      RawSpan
+	tracer     *tracerImpl
+	sync.Mutex // protects the fields below
+	raw        RawSpan
+	// TODO(tschottdorf): should this be available to the Recorder
```

Comment:
Ok. Don't think that was the case before, but I can fix it. I'll put it on RawSpan if that's ok.